Packages/com.unity.inputsystem/InputSystem/Devices/Keyboard.cs

```diff
@@ -779,4 +777,72 @@ public void OnIMECompositionChanged(IMECompositionString compositionString)
 
         internal InlinedArray<Action<IMECompositionString>> m_ImeCompositionListeners;
     }
+
+    public static class KeyboardExtensions
+    {
+        public static bool IsModifierKey(this Key key)
+        {
+            switch (key)
+            {
+                case Key.LeftAlt:
+                case Key.RightAlt:
+                case Key.LeftShift:
+                case Key.RightShift:
+                case Key.LeftMeta:
+                case Key.RightMeta:
+                case Key.LeftCtrl:
+                case Key.RightCtrl:
```

Comment:
Is Caps Lock a modifier key? Is "Fn" (guess we don't actually support Fn atm. Is there a reason for this)?

Reply:
Caps Lock is not usually considered a modifier key as far as I'm aware.

AFAIK Fn is not reported by OSs. My assumption is that in many cases the key is actually handled by the keyboard's HW itself. In any way, responding to Fn doesn't seem like a useful thing for Unity apps.

Reply:
Finally got it :) Fixed in https://github.com/Unity-Technologies/InputSystem/pull/508/commits/91a4115cd7d7d8add591985c4799d942e2e317be.